src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestResultAvailable/TestResultAvailableViewModel.swift

```diff
@@ -37,6 +37,7 @@ final class TestResultAvailableViewModel {
 	lazy var navigationFooterItem: ENANavigationFooterItem = {
 		let item = ENANavigationFooterItem()
 
+		item.hidesBackButton = true
```

Comment:
Will dis disable the back swipe gesture as well?

Reply:
I just tried it, seems like the gesture is deactivated as well 👍